src/openloco/window.cpp

```diff
@@ -365,6 +365,13 @@ namespace openloco::ui
         return scrollIndex;
     }
 
+    void window::set_disabled_widgets_and_invalidate()
+    {
+        registers regs;
+        regs.esi = (int32_t)this;
+        call(0x004CC7CB, regs);
```

Comment:
This takes `eax` as a parameter as well

Reply:
You're right. The call in the toolbar's resize event sets `eax` to 0 though. We can pass a nullpointer, though.